site/includes/footer.php

```diff
@@ -1,6 +1,34 @@
-        <footer class="bg-dark">
-            <p class="h2 mb-4">Open Game Data</p>
-            <img class="logo" src="assets/images/logos/FieldDay-logo.png" alt="Field Day">
+        <footer>
+            <div class="row">
+                <div id="footer-ogd" class="col mt-5 mb-5">
```

Comment:
With the larger amount of content, I think it works better to wrap below the medium breakpoint, so change `col` to `col-md`